### PR TITLE
Domain Name - Ionos > OVHcloud incoming transfer - JE EMLyon

### PR DESCRIPTION
**Pull request by UsernameCharbelisnotavailable**

### CODE REVIEW

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: 'Transférer un nom de domaine Ionos vers OVHcloud'
+excerpt: 'Vous trouverez dans ce guide différentes informations concernant le transfert d’un nom de domaine Ionos vers OVHcloud'
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
excerpt: 'Découvrez différentes informations concernant le transfert d’un nom de domaine Ionos vers OVHcloud'
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+updated: 2024-03-22
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
updated: 2024-03-25
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+---
+
+## Objectif
+
+Le transfert d'un nom de domaine Ionos nécessite de respecter une démarche spécifique.
+
+**Découvrez comment transférer un nom de domaine Ionos vers OVHcloud**
+
+> [!warning]
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md:13]:
```suggestion
> [!warning]
>
> Le *bureau d'enregistrement* d'un nom de domaine représente l'organisation/fournisseur agréé auprès duquel le nom de domaine est enregistré/souscrit par un particulier, une association ou une organisation. C'est auprès de ce même *bureau d'enregistrement* que vous renouvelez la souscription de votre nom de domaine (généralement une fois par an).
>
> Si OVHcloud est déjà le *bureau d'enregistrement* de votre nom de domaine **avant** de démarrer la procédure qui va suivre, le *transfert entrant de domaine* n'est pas la procédure appropriée. La procédure de *transfert entrant de domaine* s’applique **uniquement** aux noms de domaine enregistrés dans un autre *bureau d'enregistrement* qu'OVHcloud.
>
> Pour transférer la gestion de votre nom de domaine vers un autre compte client OVHcloud, la méthode adéquate est un **changement de contacts**. La procédure est décrite dans [ce guide](/pages/account_and_service_management/account_information/managing_contacts).
>
> Si vous devez également changer le **propriétaire** du nom de domaine, vous devez le faire **avant** de changer les contacts du nom de domaine. Pour cela, suivez les instructions décrites dans notre documentation sur le [changement de propriétaire des noms de domaine](/pages/web_cloud/domains/trade_domain).
>
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md:15]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+> Si le nom de domaine en cours de modification est actuellement enregistré chez OVHcloud, le transfert entrant de domaine n'est pas la procédure appropriée. Cette procédure s’applique uniquement au changement de nom de domaine enregistré (OVHcloud).
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md:17]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+> Pour transférer la gestion de votre nom de domaine vers un autre compte client OVHcloud, la méthode adéquate est un **changement de contacts**. La procédure est décrite dans [ce guide](/pages/account_and_service_management/account_information/managing_contacts).
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md:18]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+> Si vous devez également changer le **propriétaire** du nom de domaine, vous devez le faire **avant** de changer les contacts du nom de domaine. Pour cela, suivez les instructions décrites dans notre documentation sur le [changement de propriétaire des noms de domaine](/pages/web_cloud/domains/trade_domain).
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md:19]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md:20]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+> Nous mettons à votre disposition ce guide afin de vous accompagner au mieux sur des tâches courantes. Néanmoins, nous vous recommandons de faire appel à un [prestataire spécialisé](https://partner.ovhcloud.com/fr/) et/ou de contacter l'éditeur du service si vous éprouvez des difficultés. En effet, nous ne serons pas en mesure de vous fournir une assistance. Plus d'informations dans la section [Aller plus loin](#aller-plus-loin) de ce guide.
+>
+
+## Prérequis
+
+- Le domaine doit être enregistrés à votre nom depuis au moins 60 jours avant que vous puissiez les transférer.
+- Le domaine ne doit pas être bloqué chez votre registrar. 
+- Vous devrez obtenir le code d'autorisation qui sera envoyé à l'adresse email du propriétaire. 
+
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion

> [!warning]
>
> OVHcloud met à votre disposition des services dont la configuration, la gestion et la responsabilité vous incombent. Il vous revient de ce fait d'en assurer le bon fonctionnement.
>
> Nous mettons à votre disposition ce guide afin de vous accompagner au mieux sur des tâches courantes. Néanmoins, nous vous recommandons de faire appel à un [prestataire spécialisé](https://partner.ovhcloud.com/fr/) et/ou de contacter votre bureau d'enregistrement actuel si vous éprouvez des difficultés. En effet, nous ne serons pas en mesure de vous fournir une assistance. Plus d'informations dans la section [Aller plus loin](#go-further) de ce guide.
>
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+> [!warning]
+>
+> Si le nom de domaine en cours de modification est actuellement enregistré chez OVHcloud, le transfert entrant de domaine n'est pas la procédure appropriée. Cette procédure s’applique uniquement au changement de nom de domaine enregistré (OVHcloud).
+>
+> Pour transférer la gestion de votre nom de domaine vers un autre compte client OVHcloud, la méthode adéquate est un **changement de contacts**. La procédure est décrite dans [ce guide](/pages/account_and_service_management/account_information/managing_contacts).
+>
+> Si vous devez également changer le **propriétaire** du nom de domaine, vous devez le faire **avant** de changer les contacts du nom de domaine. Pour cela, suivez les instructions décrites dans notre documentation sur le [changement de propriétaire des noms de domaine](/pages/web_cloud/domains/trade_domain).
+>
+
+> [!warning]
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md:34]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+>
+> Si le nom de domaine en cours de modification est actuellement enregistré chez OVHcloud, le transfert entrant de domaine n'est pas la procédure appropriée. Cette procédure s’applique uniquement au changement de nom de domaine enregistré (OVHcloud).
+>
+> Pour transférer la gestion de votre nom de domaine vers un autre compte client OVHcloud, la méthode adéquate est un **changement de contacts**. La procédure est décrite dans [ce guide](/pages/account_and_service_management/account_information/managing_contacts).
+>
+> Si vous devez également changer le **propriétaire** du nom de domaine, vous devez le faire **avant** de changer les contacts du nom de domaine. Pour cela, suivez les instructions décrites dans notre documentation sur le [changement de propriétaire des noms de domaine](/pages/web_cloud/domains/trade_domain).
+>
+
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md:21]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+> [!warning]
+>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md:35]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+> OVHcloud met à votre disposition des services dont la configuration, la gestion et la responsabilité vous incombent. Il vous revient de ce fait d'en assurer le bon fonctionnement.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md:36]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md:37]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 17]
```diff
@@ -0,0 +1,66 @@
...
+> Nous mettons à votre disposition ce guide afin de vous accompagner au mieux sur des tâches courantes. Néanmoins, nous vous recommandons de faire appel à un [prestataire spécialisé](https://partner.ovhcloud.com/fr/) et/ou de contacter l'éditeur du service si vous éprouvez des difficultés. En effet, nous ne serons pas en mesure de vous fournir une assistance. Plus d'informations dans la section [Aller plus loin](#aller-plus-loin) de ce guide.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 18]
```diff
@@ -0,0 +1,66 @@
...
+>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md:39]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 19]
```diff
@@ -0,0 +1,66 @@
...
+
+## Prérequis
+
+- Le domaine doit être enregistrés à votre nom depuis au moins 60 jours avant que vous puissiez les transférer.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
- Le nom de domaine est enregistré auprès du bureau d'enregistrement Ionos.
- Le nom de domaine existe depuis plus de 60 jours.
- Le nom de domaine n'a pas été transféré ou n'a pas changé de propriétaire au cours des 60 derniers jours.
- L'état du nom de domaine est « OK » ou « Transférable ».
- Le nom de domaine n'a pas expiré et a une date d'expiration permettant de terminer le processus de transfert dans les temps (recommandé : plus de 60 jours).
- Être en mesure de déverrouiller le nom de domaine.
- Posséder le code de transfert ou être en mesure de le récupérer.
- Être habilité à demander le transfert du nom de domaine.
- Avoir prévenu le propriétaire du nom de domaine et/ou ses administrateurs de la demande de transfert.
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 20]
```diff
@@ -0,0 +1,66 @@
...
+- Le domaine ne doit pas être bloqué chez votre registrar.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 21]
```diff
@@ -0,0 +1,66 @@
...
+> OVHcloud met à votre disposition des services dont la configuration, la gestion et la responsabilité vous incombent. Il vous revient de ce fait d'en assurer le bon fonctionnement.
+>
+> Nous mettons à votre disposition ce guide afin de vous accompagner au mieux sur des tâches courantes. Néanmoins, nous vous recommandons de faire appel à un [prestataire spécialisé](https://partner.ovhcloud.com/fr/) et/ou de contacter l'éditeur du service si vous éprouvez des difficultés. En effet, nous ne serons pas en mesure de vous fournir une assistance. Plus d'informations dans la section [Aller plus loin](#aller-plus-loin) de ce guide.
+>
+
+## Prérequis
+
+- Le domaine doit être enregistrés à votre nom depuis au moins 60 jours avant que vous puissiez les transférer.
+- Le domaine ne doit pas être bloqué chez votre registrar. 
+- Vous devrez obtenir le code d'autorisation qui sera envoyé à l'adresse email du propriétaire.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 22]
```diff
@@ -0,0 +1,66 @@
...
+
+Pour trouver le code d'autorisation de votre domaine, accédez à la section "Domaines" sur la page [Transfert & renouvellement]([[https://www.nominet.uk/whois/](https://login.ionos.fr/?redirect_url=https%3A%2F%2Fmy.ionos.fr%2Fstart-with-domain%2Fprivacy%3Futm_source%3Dhelpcenter%26utm_medium%3Dknowledge%26utm_campaign%3D218%26utm_term%3Dprivacy%26utm_content%3Ddeeplink%26skipIntcpts%3Dtrue](https://login.ionos.fr/?redirect_url=https%3A%2F%2Fmy.ionos.fr%2Fstart-with-domain%2Ftransfer%3Futm_source%3Dhelpcenter%26utm_medium%3Dknowledge%26utm_campaign%3D218%26utm_term%3Dtransfer%26utm_content%3Ddeeplink%26skipIntcpts%3Dtrue))){.external}. Sélectionnez votre domaine, puis cliquez sur **Afficher le code d'autorisation**.
+
+Une fois le code d'authorisation obtenu, vous pouvez procéder au transfère de domaine en suivant les étapes de notre guide [Transférer son nom de domaine vers OVHcloud](/pages/web_cloud/domains/transfer_incoming_generic_domain).
+
+## Aller plus loin <a name="aller-plus-loin"></a>
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
## Aller plus loin <a name="go-further"></a>
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 23]
```diff
@@ -0,0 +1,66 @@
...
+
+- Le domaine doit être enregistrés à votre nom depuis au moins 60 jours avant que vous puissiez les transférer.
+- Le domaine ne doit pas être bloqué chez votre registrar. 
+- Vous devrez obtenir le code d'autorisation qui sera envoyé à l'adresse email du propriétaire. 
+
+## En pratique
+
+### Désactiver l'enregistrement privé
+
+Pour confirmer si l'enregistrement privé est activé pour votre domaine, veuillez accéder à la section **Domaines** sur la page [Confidentialité & données de contact]([https://www.nominet.uk/whois/](https://login.ionos.fr/?redirect_url=https%3A%2F%2Fmy.ionos.fr%2Fstart-with-domain%2Fprivacy%3Futm_source%3Dhelpcenter%26utm_medium%3Dknowledge%26utm_campaign%3D218%26utm_term%3Dprivacy%26utm_content%3Ddeeplink%26skipIntcpts%3Dtrue)){.external}. Choisissez votre domaine, puis désactivez l'option d'enregistrement privé. Si cette option ne figure pas sur la page, aucune action est requise et cela signifie que la désactivation de l'enregistrement privé n'est pas disponible pour votre domaine.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
Pour confirmer si l'enregistrement privé est activé pour votre nom de domaine, accédez à la section `Domaines`{.action} sur la page « [Confidentialité & données de contact](https://login.ionos.fr/?redirect_url=https%3A%2F%2Fmy.ionos.fr%2Fstart-with-domain%2Fprivacy%3Futm_source%3Dhelpcenter%26utm_medium%3Dknowledge%26utm_campaign%3D218%26utm_term%3Dprivacy%26utm_content%3Ddeeplink%26skipIntcpts%3Dtrue)){.external} ». Choisissez votre nom de domaine, puis désactivez l'option d'enregistrement privé. Si cette option ne figure pas sur la page, aucune action n'est requise et cela signifie que la désactivation de l'enregistrement privé est indisponible pour votre nom de domaine.
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 24]
```diff
@@ -0,0 +1,66 @@
...
+
+### Désactiver le verrouillage du transfert
+
+> [!warning]
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md:49]:
```suggestion
> [!warning]
>
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 25]
```diff
@@ -0,0 +1,66 @@
...
+>Par mesure de sécurité, les domaines avec une extension générique (par exemple .com, .net ou .org) sont automatiquement verrouillés par
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
> Par sécurité, les noms de domaine avec une extension générique (par exemple *.com*, *.net* ou *.org*) sont verrouillés par défaut et ne peuvent pas être transférés. Avant d'initier un transfert de nom de domaine, vous devez désactiver ce verrouillage de transfert.
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 26]
```diff
@@ -0,0 +1,66 @@
...
+
+### Désactiver l'enregistrement privé
+
+Pour confirmer si l'enregistrement privé est activé pour votre domaine, veuillez accéder à la section **Domaines** sur la page [Confidentialité & données de contact]([https://www.nominet.uk/whois/](https://login.ionos.fr/?redirect_url=https%3A%2F%2Fmy.ionos.fr%2Fstart-with-domain%2Fprivacy%3Futm_source%3Dhelpcenter%26utm_medium%3Dknowledge%26utm_campaign%3D218%26utm_term%3Dprivacy%26utm_content%3Ddeeplink%26skipIntcpts%3Dtrue)){.external}. Choisissez votre domaine, puis désactivez l'option d'enregistrement privé. Si cette option ne figure pas sur la page, aucune action est requise et cela signifie que la désactivation de l'enregistrement privé n'est pas disponible pour votre domaine.
+
+### Désactiver le verrouillage du transfert
+
+> [!warning]
+>Par mesure de sécurité, les domaines avec une extension générique (par exemple .com, .net ou .org) sont automatiquement verrouillés par 
+>défaut et ne peuvent pas être transférés. Avant d'initier un transfert de domaine, vous devez désactiver ce verrouillage de transfert.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
>
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 27]
```diff
@@ -0,0 +1,66 @@
...
+
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 28]
```diff
@@ -0,0 +1,66 @@
...
+
+Pour vérifier si le verrouillage de transfert de domaine est activé pour votre domaine, veuillez accéder à la section "Domaines" sur la page "Transfert & renouvellement". Sélectionnez ensuite votre domaine et désactivez l'option de verrouillage de transfert de domaine en utilisant le curseur.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
Pour vérifier si le verrouillage de transfert est activé pour votre nom de domaine, rendez-vous sur la page `Transfert & renouvellement`{.action}, puis accédez à la section `Domaines`{.action}. Sélectionnez ensuite votre nom de domaine et désactivez l'option de verrouillage de transfert de domaine en utilisant le curseur.
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 29]
```diff
@@ -0,0 +1,66 @@
...
+
+### Obtenir le code d'autorisation
+
+Le code d'autorisation sécurise votre domaine contre tout transfert non autorisé par des tiers. Ce code est requis pour autoriser le transfert de votre domaine vers un nouveau fournisseur.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
Le code d'autorisation protège votre nom de domaine contre tout transfert non autorisé par des tiers. Ce code est requis pour autoriser le transfert de votre nom de domaine vers un nouveau fournisseur.
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 30]
```diff
@@ -0,0 +1,66 @@
...
+
+Pour trouver le code d'autorisation de votre domaine, accédez à la section "Domaines" sur la page [Transfert & renouvellement]([[https://www.nominet.uk/whois/](https://login.ionos.fr/?redirect_url=https%3A%2F%2Fmy.ionos.fr%2Fstart-with-domain%2Fprivacy%3Futm_source%3Dhelpcenter%26utm_medium%3Dknowledge%26utm_campaign%3D218%26utm_term%3Dprivacy%26utm_content%3Ddeeplink%26skipIntcpts%3Dtrue](https://login.ionos.fr/?redirect_url=https%3A%2F%2Fmy.ionos.fr%2Fstart-with-domain%2Ftransfer%3Futm_source%3Dhelpcenter%26utm_medium%3Dknowledge%26utm_campaign%3D218%26utm_term%3Dtransfer%26utm_content%3Ddeeplink%26skipIntcpts%3Dtrue))){.external}. Sélectionnez votre domaine, puis cliquez sur **Afficher le code d'autorisation**.
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
Pour trouver le code d'autorisation de votre nom de domaine, rendez-vous sur la page « [Transfert & renouvellement`(https://login.ionos.fr/?redirect_url=https%3A%2F%2Fmy.ionos.fr%2Fstart-with-domain%2Ftransfer%3Futm_source%3Dhelpcenter%26utm_medium%3Dknowledge%26utm_campaign%3D218%26utm_term%3Dtransfer%26utm_content%3Ddeeplink%26skipIntcpts%3Dtrue){.external} », puis accédez à la section `Domaines`{.action}. Sélectionnez votre nom de domaine, puis cliquez sur `Afficher le code d'autorisation`{.action}.
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 31]
```diff
@@ -0,0 +1,66 @@
...
+
+Une fois le code d'authorisation obtenu, vous pouvez procéder au transfère de domaine en suivant les étapes de notre guide [Transférer son nom de domaine vers OVHcloud](/pages/web_cloud/domains/transfer_incoming_generic_domain).
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
Une fois le code d'autorisation obtenu, vous pouvez procéder au transfert de votre nom de domaine en suivant les étapes de notre guide « [Transférer son nom de domaine vers OVHcloud](/pages/web_cloud/domains/transfer_incoming_generic_domain) ».
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 32]
```diff
@@ -0,0 +1,66 @@
...
+
+## Aller plus loin <a name="aller-plus-loin"></a>
+
+[Migration de votre site Web et de vos e-mails vers OVHcloud](/pages/web_cloud/web_hosting/hosting_migrating_to_ovh)
```
Comment (Kazarus4) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
[Migrer son site Web et ses e-mails vers OVHcloud](/pages/web_cloud/web_hosting/hosting_migrating_to_ovh)
```

**Review by Kazarus4 — COMMENTED** [review]
FRFR proof done

File: pages/index.md [hunk 33]
```diff
@@ -1028,6 +1028,7 @@
                 + [Transferring a domain name to another registrar](web_cloud/domains/transfer_outgoing_domain)
                 + [Transferring a .uk domain name to another registrar](web_cloud/domains/transfer_outgoing_couk)
                 + [Transferring a domain name to OVHcloud](web_cloud/domains/transfer_incoming_generic_domain)
+                + [Transfering a domain name from Ionos to OVHcloud](web_cloud/domains/transfer_incoming_ionos)
```
Comment (OVH-Team-Guides) [inline pages/index.md]:
```suggestion
                + [Transferring a domain name from Ionos to OVHcloud](web_cloud/domains/transfer_incoming_ionos)
```

File: pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md [hunk 34]
```diff
@@ -0,0 +1,72 @@
...
+- Être en mesure de déverrouiller le nom de domaine.
+- Posséder le code de transfert ou être en mesure de le récupérer.
+- Être habilité à demander le transfert du nom de domaine.
+- Avoir prévenu le propriétaire du nom de domaine et/ou ses administrateurs de la demande de transfert.
+
+> [!warning]
+>
+> OVHcloud met à votre disposition des services dont la configuration, la gestion et la responsabilité vous incombent. Il vous revient de ce fait d'en assurer le bon fonctionnement.
+>
+> Nous mettons à votre disposition ce guide afin de vous accompagner au mieux sur des tâches courantes. Néanmoins, nous vous recommandons de faire appel à un [prestataire spécialisé](https://partner.ovhcloud.com/fr/) et/ou de contacter votre bureau d'enregistrement actuel si vous éprouvez des difficultés. En effet, nous ne serons pas en mesure de vous fournir une assistance. Plus d'informations dans la section [Aller plus loin](#go-further) de ce guide.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/transfer_incoming_ionos/guide.fr-fr.md]:
```suggestion
> Nous mettons à votre disposition ce guide afin de vous accompagner au mieux sur des tâches courantes. Néanmoins, nous vous recommandons de faire appel à un [prestataire spécialisé](https://partner.ovhcloud.com/fr/directory/) et/ou de contacter votre bureau d'enregistrement actuel si vous éprouvez des difficultés. En effet, nous ne serons pas en mesure de vous fournir une assistance. Plus d'informations dans la section [Aller plus loin](#go-further) de ce guide.
```

**Pull request merged**